### ISSUE TITLE
[Sourcing][Accessibility] fdp-table role="rowheader" missing for row's heading text 

### ISSUE THREAD

**Issue by I543348** [SAP/fundamental-ngx#9536]
#### Is this a bug, enhancement, or feature request?
Bug

#### Briefly describe your proposal.
role=rowheader is missing for table rows in fdp-table.
As per w3 standards , upon changing row using keyboard the row number and role has to be readout by screenreader
https://www.w3.org/TR/wai-aria-1.0/roles#rowheader 

#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)
angular 15
core / platform: 0.39.1-rc-5

#### If this is a bug, please provide steps for reproducing it.
this is replicable on simple table in https://sap.github.io/fundamental-ngx/#/platform/table but partially works on https://sap.github.io/fundamental-ngx/#/platform/table#actiable-rows (row number is read out here )

**Comment by fkolar** [comment]
I have checked also across all the SAPUI5 tables and rowheader is not present.  The same goes for UI5 WebComponents. 
https://sapui5.hana.ondemand.com/#/entity/sap.ui.webc.main.Table/sample/sap.ui.webc.main.sample.Table

https://sapui5.hana.ondemand.com/#/entity/sap.ui.table.Table/sample/sap.ui.table.sample.Basic


Should we close this? I have also noticed that in 2021 UI5 team did a few changes removing this from their code.

**Comment by I543348** [comment]
Thanks @fkolar , i shall check with ATL team and get back on this.

**Comment by simlin** [comment]
Yes, indeed. In the past we used the role rowheader for the selection column in the sap.ui.table.Table. After feedback from our ACC experts we changed that, because the role should only be used when the cell labels the row. This is not the case for the selection.

**Comment by fkolar** [comment]
@simlin, do you think you can help me understand the "when the cell labels the row", do you have some example?  
Still not sure if it should be used or not.

**Comment by simlin** [comment]
@fkolar according to our ACC colleagues a column header is something like in the picture below. In the UI5 tables we don not have such a mechanismn: 

![image](https://user-images.githubusercontent.com/9136757/228516705-ec0f7b92-8f96-4d58-8054-5931e7f574f9.png)

**Comment by fkolar** [comment]
Make sense, @I543348, do we have such case ? In the above like its not like in the picture. These is more PIVOT layout

**Comment by I543348** [comment]
we do have such cases for guided sourcing application.
DTAG customer has reported this issue for Monitor supplier bids table 
<img width="876" alt="Screenshot 2023-03-29 at 8 37 17 PM" src="https://user-images.githubusercontent.com/88182358/228583447-5581b2cf-3506-445e-81c5-335f04e6c0db.png">

**Comment by I543348** [comment]
@simlin , can you please take a look at the above comment's screenshot , and confirm if role=rowheader is applicable here or not?

Thanks.

**Comment by fkolar** [comment]
There is no way to get around this. As already mention this goes outside of fundamental-table scope. I double check no workaround would work to get down on the TD level and provide this attribute.

**Comment by simlin** [comment]
> [I543348](/I543348)

In UI5 the column with the tree icons are also implemented as "normal" cells and not as row headers

**Comment by I543348** [comment]
Thanks @simlin and @fkolar

**Comment by OliKei** [comment]
In case the user uses the Jaws screen reader, there are shortcuts which describe row contexts: JAWS+SHIFT+UP or ALT+SHIFT+COMMA on notebook keyboard layouts.
To read all content up to current cell: JAWS+SHIFT+HOME.

With Voice Over VO+C or VO+R might help.

With NVDA use NVDA+Numpad 8 to move reading focus one or more levels up, then press NVDA+Numpad 4 to move the reading focus to previous columns and the reader should read the column‘s content without changing the keyboard focus.

Does this help to get context information on the current row?
